weaver/core/relay/config/Corda_Relay.toml

```diff
@@ -5,6 +5,7 @@ hostname="localhost"
 db_path="db/Corda_Relay/requests"
 # This will be replaced by the task queue.
 remote_db_path="db/Corda_Relay/remote_request"
+db_satp_path="db/Corda_Relay"
```

Comment:
Can you add a prefix to the db path say `db/Corda_Relay/satp`, so that all satp db related files are in one directory?
That's how we planned to structure the `db/Corda_Relay` directory, it contains one directory for each db.

Please apply same the change in other toml files too.

Reply:
Done